pgoapi/pgoapi.py

```diff
@@ -519,6 +524,19 @@ def get_caught_pokemons(self, inventory_items):
                 pokemon_data = inventory_item['inventory_item_data']['pokemon_data']
                 pokemon = Pokemon(pokemon_data, self.pokemon_names, self.game_master.get(pokemon_data.get('pokemon_id', 0), PokemonData()))
 
+                # Need to calculate score here since we want to read in values from config file
+                if self.USE_CUSTOM_SCORE_EXPRESSION:
+                    CP = pokemon.cp
+                    MAX_CP = pokemon.max_cp
+                    IV = pokemon.iv
+                    try:
+                        pokemon.score = eval(self.CUSTOM_SCORE_EXPRESSION)
+                    except Exception as e:
+                        self.log.error("Error in evaluating custom score expression %s", e)
+                else:
+                    if pokemon.max_cp > 0 and pokemon.iv > 0:
+                        pokemon.score = pokemon.iv / 100.0 * self.SCORE_IV_WEIGHT + pokemon.cp / pokemon.max_cp * self.SCORE_CP_WEIGHT
+
```

Comment:
I would indeed like to do that. Would you help me figure out the best way to pass those config constants into pokemon.py? I'm not a Python convention expert by any means, but I felt like passing constants through the Pokemon initializer seemed like a bad idea.